server/src/test/scala/za/co/absa/atum/server/api/controller/PartitioningControllerIntegrationTests.scala

```diff
@@ -27,18 +27,30 @@ import zio.test._
 
 object PartitioningControllerIntegrationTests extends ZIOSpecDefault with TestData {
```

Comment:
Please rename the class. It's not an integration test and therefore should be executed as a unit test.